src/components/CollateralChoice.tsx

```diff
@@ -75,8 +98,18 @@ const CollateralChoice = ({ id }: CollateralChoiceParams) => {
   // https://github.com/Agoric/agoric-sdk/issues/6518
   const collateralTitle = 'ATOM';
 
+  const isSelected = selectedCollateralId === id;
+
   return (
-    <div className="w-fit px-6 pt-2 pb-4 bg-white rounded-[10.7px] shadow-[0_25px_35.6px_0_rgba(116,116,116,0.25)]">
+    <button
```

Comment:
[Buttons can have content](https://css-tricks.com/use-button-element/#aa-buttons-can-have-content) but the table doesn't seems to fit the [content categories of a button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#technical_summary).

What motivates this? I see there's an `onClick` handler so that would work on a `div` too.

Reply:
Hmm yea it needs to be clickable because you click this card to select your collateral choice. I initially did a div with a click handler but the linter told me divs are non-interactive. So I could either give it role="button", and implement the keyboard listeners myself, or just make it a button. We might just have to break the rules on this one.

Reply:
fair. please leave a comment with the rationale